### PR TITLE
Stop then start oauth2-service after start up

### PR DESCRIPTION
**Pull request by benjefferies**
* Stop and start oauth as it falls over if it comes up before DB but the
  process doesn't die

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 up:
-	docker-compose -f dev.yml -f ras-services.yml -f rm-services.yml up -d ${SERVICE}
+	docker-compose -f dev.yml -f ras-services.yml -f rm-services.yml up -d
+up_hack:
```
Comment (benjefferies) [inline Makefile]: Suggestions welcome on the name

**Comment by insacuri** [comment]
Not sure how much work it would take but wouldn't the more correct option be to update the oauth service to not die if the db isn't immediately there?

**Comment by benjefferies** [comment]
@insacuri 100% that's the right thing to do. I should be able to do this for the non django webservices but I haven't found a way to do this with django 👎

**Comment by andrewmil** [comment]
Can we not specify that the oauth service is dependent on postres in the docker-compose files? Or can we not do that because they're set up in different files?

**Comment by benjefferies** [comment]
Docker doesn't really work like that @andrewmil it relies on you sorting out the dependencies yourself. Described here https://docs.docker.com/compose/startup-order/

**Pull request merged**